src/AutoRest.CSharp/Mgmt/AutoRest/MgmtOutputLibrary.cs

```diff
@@ -217,30 +217,34 @@ public MgmtOutputLibrary(CodeModel codeModel, BuildContext<MgmtOutputLibrary> co
             return _armResource;
         }
 
-        private Dictionary<Operation, LongRunningOperation> EnsureLongRunningOperations()
+        private Dictionary<Operation, MgmtLongRunningOperation> EnsureLongRunningOperations()
         {
             if (_longRunningOperations != null)
             {
                 return _longRunningOperations;
             }
 
-            _longRunningOperations = new Dictionary<Operation, LongRunningOperation>();
+            _longRunningOperations = new Dictionary<Operation, MgmtLongRunningOperation>();
 
             if (_context.Configuration.PublicClients)
             {
                 foreach (var operationGroup in _codeModel.OperationGroups)
                 {
-                    foreach (var operation in operationGroup.Operations)
+                    // if non-resource, we won't be able to get the info for LRO
+                    if (operationGroup.IsResource(_mgmtConfiguration))
```

Comment:
if its not a resource the LRO should look like the base, but they should still be here.

Reply:
No doubt. By not doing the check those missing LRO objects are back.